### PR TITLE
Edits to installs and consistent nomenclature for versions

### PR DESCRIPTION
**Pull request by ebgitelman**
Part I of install topics transferred from outdated branch.

Also made xx nomenclature consistent throughout the doc, as it was using three different styles of wording.

### CODE REVIEW

**Comment by ebgitelman** [comment]
This pull request includes changes to install topics and also includes changes to the nomenclature surrounding xx. This was implemented inconsistently throughout the doc, so I made them consistent. 

@nidhibhammar You might want to look at the graphical install topic to see if I changed any steps, but I think it's okay. I revised that topic to have step numbers as well as comply to our UI standards.

I will revise the Stackbuilder content again after this pull request is approved so I can add step numbers to that as well.

File: product_docs/docs/epas/15/installing/uninstalling/linux_uninstall.mdx [hunk 1]
```diff
@@ -53,7 +53,7 @@ To uninstall EDB Postgres Advanced Server and its dependent packages, use the ap
     ```text
     yum remove edb-as<xx>-server*
     ```
-Where `<xx>` represents the EDB Postgres Advanced Server version.
+Where `<xx>` is EDB Postgres Advanced Server version number.
```
Comment (francoughlin) [inline product_docs/docs/epas/15/installing/uninstalling/linux_uninstall.mdx:56]: Is there an article missing here?

**Review by francoughlin — APPROVED** [review]
Looks great! One minor nit.

File: product_docs/docs/epas/15/installing/uninstalling/linux_uninstall.mdx (hunk 1 above)
Comment (nidhibhammar) [inline product_docs/docs/epas/15/installing/uninstalling/linux_uninstall.mdx:56]:
```suggestion
Where `<xx>` is the EDB Postgres Advanced Server version number.
```

File: product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/performing_a_graphical_installation_on_windows.mdx [hunk 2]
```diff
@@ -7,240 +7,200 @@ redirects:
...
+    If you select **Compatible with PostgreSQL**, EDB Postgres Advanced Server exhibits compatibility with PostgreSQL version 15. If you choose to install in Compatible with PostgreSQL mode, the default EDB Postgres Advanced Server superuser name is postgres. For detailed information about PostgreSQL functionality, see the [PostgreSQL website](http://www.postgresql.org).
 
-Accept the default file locations, or use the `File Browser` icon to select an alternate location; click `Next` to continue to the `EPAS Dialect` window.
+1. After specifying a configuration mode, select **Next**. The Password window opens.
 
-![The EDB Postgres Advanced Server Dialect window](../../../images/advanced_server_dialect.png)
+    ![The Password window](../../../images/password_window.png)
 
-<div style="text-align: center"> Fig. 6: The EDB Postgres Advanced Server Dialect window </div>
+    EDB Postgres Advanced Server uses the password specified on the Password window for the database superuser. The specified password must conform to any security policies on the EDB Postgres Advanced Server host.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/performing_a_graphical_installation_on_windows.mdx:140]:
```suggestion
    EDB Postgres Advanced Server uses the password specified on the Password window for the database superuser. The specified password must confirm to any security policies on the EDB Postgres Advanced Server host.
```

**Review by nidhibhammar — APPROVED** [review]

**Comment by nidhibhammar** [comment]
There are some warnings related to overlap of redirects which need to be fixed.

Screenshot 2023-08-21 at 12 08 55 PM

**Pull request merged**